.github/workflows/ci-nym-vpn-cli.yml

```diff
@@ -100,29 +102,84 @@ jobs:
           echo "TRIPLET=$triplet" >> $GITHUB_ENV
           echo "RUSTFLAGS=-L ${GITHUB_WORKSPACE}/build/lib/${triplet}" >> $GITHUB_ENV
 
+      - name: Set extra env for Windows
+        if: contains(matrix.os, 'windows')
+        shell: bash
+        run: |
+          echo "RUSTFLAGS=-L ${GITHUB_WORKSPACE}/build/lib/x86_64-pc-windows-msvc -L ${GITHUB_WORKSPACE}/build/lib/ -Clink-args=/LIBPATH:${GITHUB_WORKSPACE}/build/lib/x64-Debug" >> $GITHUB_ENV
+          mkdir -p ${GITHUB_WORKSPACE}/build/lib/
+
       - name: Install Go
-        uses: actions/setup-go@v4
+        uses: actions/setup-go@v5
         with:
-          go-version: 'stable'
+          go-version: "stable"
 
       - name: Install Protoc
         uses: arduino/setup-protoc@v2
         if: contains(matrix.os, 'macos') || contains(matrix.os, 'windows')
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: rustfmt check
-        run: cargo fmt --check
+      - name: Setup MSBuild.exe
+        if: contains(matrix.os, 'windows')
+        uses: microsoft/setup-msbuild@v2
 
       - name: Build wireguard
-        run: ./wireguard/build-wireguard-go.sh
+        shell: bash
+        run: |
+          ./wireguard/build-wireguard-go.sh
 
       - name: Build wireguard (Android)
         if: contains(matrix.target, 'android')
         run: ./wireguard/libwg/build-android.sh
 
+      - name: Checkout mullvad libs for Windows
+        if: contains(matrix.os, 'windows')
+        uses: actions/checkout@v4
+        with:
+          repository: nymtech/nym-vpn-mullvad-libs
```

Comment:
I think we need to checkout the `nym_patches` branch

Reply:
It works as is, I can change if needed.